apps/condo/domains/common/components/PhoneLink.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { formatPhone, stripPhone } from '../utils/helpers'
+import { PHONE_CLEAR_REGEXP } from '../constants/regexps'
+
+interface IPhoneLink {
+    value: string
+}
+
+export const PhoneLink: React.FC<IPhoneLink> = ({ value }) => (
+    <a href={`tel:${stripPhone(value)}`}>
```

Comment:
it's better to use already exists `normalizePhone` function

Reply:
Used in `fb03c61c`